src/script/descriptor.cpp

```diff
@@ -2,6 +2,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include "attributes.h"
```

Comment:
Causes [linter failure](https://github.com/bitcoin/bitcoin/pull/31734/checks?check_run_id=36770622181):
```
[08:12:43.311] Please use bracket syntax includes ("#include <foo.h>") instead of quote syntax includes:
[08:12:43.311] src/script/descriptor.cpp:#include "attributes.h"
```
```suggestion
#include <attributes.h>
```

Reply:
my bad, my clangd add it automatically and I don't notice.
I drop it as it seems redundant